Cinebase/Use Cases/Movies/MovieDetail/Showtimes/ShowtimesViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ShowtimesViewController.swift
+//  Cinebase
+//
+//  Created by Charles Moncada on 10/05/17.
+//  Copyright © 2017 Globant. All rights reserved.
+//
+
+import UIKit
+
+
+// MARK: ShowtimesView Protocol Definition
+protocol ShowtimesView: class {
+    func handleBuyTicketsButtonPressed(_ sender: UITableViewCell)
```

Comment:
This protocol is meant to be used (called) by the presenter. In that sense, it doesn't make sense for the presenter to pass the cell as a parameter because it doesn't have any reference from UIKit